### ISSUE TITLE
Create Hero Research

### ISSUE THREAD

**Issue by jeffg2k** [StargateInvasion/SGI#14]
Add the Heros to the Defense branch as part of a research tree.  Make the Heros auto-spawn at the homeworld when the research is completed.  Update the Hero descriptions with information about the ships.

**Comment by AceFighter** [comment]
We actually have a HQ structure, so once they research the player will manually have to do this.

Also don't know if it's possible for it to autospawn once the research is complete.

**Comment by jeffg2k** [comment]
They did an autospawn in STA3, so maybe we can take a look at their method or ask them.

**Comment by jeffg2k** [comment]
Research created https://github.com/StargateInvasion/SGI/commit/5a621d34c4d688e181efa2c69f9440c5025b290a

We can reopen or create a new issue if we want to look at auto-spawn.
